### DISCUSSION TITLE
Do @formConnectedFields work in create.blade.php modal forms?

### DISCUSSION THREAD

**Discussion by ghost** [area17/twill#1208] [Q&A] [answered]
Hi,

I can only seem to get @formConnectedFields to work in form.blade.php. Are they not supported in the create.blade.php modal?

**Comment by pboivin** [comment, marked as answer]
Hey @aarontilt. That's right, not all fields are supported in the `create.blade.php` modal at the moment, but I believe this will be addressed in the near future.

For reference: https://github.com/area17/twill/issues/1050
